src/manual/design/namespaces.md

```diff
@@ -152,6 +152,16 @@ Artifact names are, by convention, slash-separated.
 * `project/<project>/…` -
    Artifact names with this prefix are the responsibility of the project, which may have further namespace conventions.
 
+* `login-identity/<identity>/…`
+   Artifact names with this prefix are the responsibility of the user with the given [login indentity](/reference/core/login).
+   This namespace enables tasks to persist data that is only readable by a single user (typically the user that submitted it), such as loaner credentials.
+
+Note, highly confidential data should not be stored in unencrpyted taskcluster
+artifacts, since they are only protected by a single (fallible) system. If
+highly sensitive data needs to be persisted, it is recommended to encrypt data
+using the public key of a key-pair whose private key has been highly secured,
+in order that at least two independent systems protect the data.
+
```

Comment:
Not sure if this was the right place to put this in the docs - I just didn't want people going to town with `login-identity/*` artifacts in tasks, but realised the principle applied to all non-public artifacts, so didn't want to make it specific to just this new namespace, but anything not under `public`.

Reply:
It makes sense.  I was going to argue with "fallible" but it's close enough.  Really the issue is that the given user is not the *only* entity with scopes that satisfy this artifact path, so it's important that you be comfortable giving all such entities access to the data in an artifact.  Those include administrative users as well as service credentials, and anything else which has been configured for access in the clients and roles.